python/mxnet/gluon/block.py

```diff
@@ -1088,6 +1088,7 @@ def _get_graph(self, *args):
                 out = super().__call__(*args)
             flatten_out, self._out_format = _flatten(out, "output")
             symbol_outputs = dc.get_symbol(flatten_out, sym_cls=type(symbol_inputs[0]))
+            dc.clear(flatten_out)
```

Comment:
Thanks, this is better than using the destructor. But I think there's still a leak of the graph and once that leak is fixed, the clear call can be removed? If so, would you mind adding a comment that this is temporary (though it's fine to keep for this PR).